vfs/vfstest/fs.go

```diff
@@ -324,7 +327,7 @@ func (r *Run) readFile(t *testing.T, filepath string) string {
 
 func (r *Run) mkdir(t *testing.T, filepath string) {
 	filepath = r.path(filepath)
-	err := r.os.Mkdir(filepath, 0700)
+	err := r.os.Mkdir(filepath, 0755)
```

Comment:
This one too :-) ?

Reply:
Directories are always created as 0755 regardless of the mode that is passed in.